launcher/src/backend/HetznerServer.js

```diff
@@ -0,0 +1,159 @@
+const log = require('electron-log');
+const https = require('https');
+
+/**
+ * Used to wait for the server to fully initialize
+ */
+function Sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+export class HetznerServer {
```

Comment:
Can we put this class on an integration test file so it's only loaded for integration tests?